django_netjsonconfig/base/admin.py

```diff
@@ -200,6 +205,15 @@ def download_view(self, request, pk):
         return send_file(filename='{0}.tar.gz'.format(config.name),
                          contents=config_archive.getvalue())
 
+    def device_context_view(self, request, pk):
+        device = get_object_or_404(self.model, pk=pk)
```

Comment:
`device` should be renamed `instance` as in the beginning, you were right on that one